### DISCUSSION TITLE
Can not inspect pinia in Vue devtools when I use the pinia 2.0.23 but 2.0.13

### DISCUSSION THREAD

**Discussion by jc-yang** [vuejs/pinia#1805] [Help and Questions]
### Reproduction

none

### Steps to reproduce the bug

When I use the latest version 2.0.23 of pinia, there is no pinia panel in Vue devtools but exists when I switched to pinia 2.0.13. The only difference of my codes is the pinia version.

### Expected behavior

Pinia option might always appear in the devtools no matter which pinia version used in my code.

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/33781521/201880041-84b9080f-ffda-4e96-9dc4-3f1debd31805.png">

### Actual behavior
When I use pinia 2.0.23
<img width="1119" alt="image" src="https://user-images.githubusercontent.com/33781521/201880471-54ccac81-9053-42df-86b8-e3b0a6229410.png">

**Comment by Pijuli** [comment]
Same. Only fix was to downgrade it. Also commented in a [closed isssue](https://github.com/vuejs/devtools/issues/1964) in Vue Devtools github. Beta version breaks even further. Proposed solution was not a fix either.
Any idea @posva ? Is downgrading the only solution for now?

**Reply by posva** [reply]
![Screenshot 2022-11-17 at 11 59 28](https://user-images.githubusercontent.com/664177/202428932-fa40b295-e819-44c3-ab68-ca810da2ab9c.png)

Can't reproduce in this repo playground so apart from upgrading dependencies, making sure you are using the latest stable release of vue devtools, or comparing projects, I can't say much more.

**Reply by Pijuli** [reply]
You are right. I can't reproduce it either with a clean vue3 install topping the versions... I have an ionic project and there I can see the router and everything loads fine but no pinia. Let's see if I have any idea on what it can be, because it is working fine except for the devtools 😢
Devtools version is the last one, btw. So no diff here

**Reply by Pijuli** [reply]
Ok, reinstalled everything and it's working now. 2.0.23 and 2.0.24, no problem whatsoever. Devtools last stable.
I don't know what happened 🤦‍♀️🤷‍♀️ Thank you, anyway. And thank you for this amazing piece of code. Really 👍
